music21/roman.py

```diff
@@ -2122,13 +2134,16 @@ def __init__(
         # immediately fix low-preference figures
         if isinstance(figure, str):
             figure = figure.replace('0', 'o')  # viio7
-
-        if isinstance(figure, str):
             # /o is just a shorthand for ø -- so it should not be stored.
             figure = figure.replace('/o', 'ø')
-
+            figure = figure.replace('º', '°')
```

Comment:
I'm not sure that I'm liking the use of degree character for diminished when seen in the context of half diminished. The two characters are supposed to be identical except for the presence of the slash, and this change moves them to be farther apart not closer. 

All else approved.

Reply:
Happy to take suggestions for what to do with the less common symbol `º` (instead of `°`). Should it raise an exception? If someone wrote `ii/º7` it would convert to `ii°7`. Is that a problem?

Reply:
OH -- is it that both `viiº` and `vii°` should go to `viio` so that it's more like `iiø7'?

Reply:
Ah, no -- my phone font was really bad at the time, and it looked like the last one was even smaller than it was.  Still I wish that the o and ø looked better.

Reply:
Yeah, thinking more and doing some edits on RomanNumeral, ° is just too small and hard to type to make it a preferred character the way ø is, so I think probably better to make viio7 the default.

Reply:
> ° is just too small and hard to type

Lol. When I need it I usually just hold down the option key and then mash the keyboard until it comes up.